ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ToDoListView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-07.
+//
+
+import SwiftUI
+
+struct ToDoSection: Identifiable {
+    var id: UUID = UUID()
+    let title: String
+    let toDoItems: [TaskModel]
+}
+struct ToDoListView: View {
+    static let items: [ToDoSection] = [
+            .init(title: "Today’s To-Do List", toDoItems: [
+                .init(title: "Cleaning", deadline: Date(), status: false),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false)
+                
+            ]),
+            .init(title: "All To-Do List", toDoItems: [
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Painting", deadline: Date(), status: true)
+            ])
+        ]
+    var body: some View {
+        NavigationStack{
+            List {
+                ForEach(ToDoListView.items) { tasks in
+                    HStack(spacing: 6) {
+                        Text(tasks.title)
+                            .foregroundColor(Color(.caption))
+                            .font(.system(size: 12))
+                            .fontWeight(.medium)
+                        Text("\(tasks.toDoItems.capacity) tasks left")
+                            .foregroundColor(.gray)
+                            .font(.system(size: 12))
+                            .fontWeight(.thin)
+                    }
+                    ForEach(tasks.toDoItems) { todo in
+                        CellTask(todo: todo)
+                    }
+                }
+                Spacer()
+            }
+            .navigationTitle("To-Do List")
+            .listStyle(.plain)
+            .padding(.leading, 16)
```

Comment:
Oh because I saw in the design, there should be a gap of size 16 between the list and the edge of the phone, without this, the list is near the edge of the phone.

Reply:
list already has default padding. if you will remove the padding you will see that it is not near the edge

Reply:
Oh ok.